File: examples/sharing/group-members.ts
import type { JsBaoClient } from "js-bao-wss-client";

// Current members of a group.
export async function listGroupMembers(
  client: JsBaoClient,
  groupType: string,
  groupId: string,
) {
  // #region example
  const members = await client.groups.listMembers(groupType, groupId);
  // [{ userId, userName, userEmail, addedAt, addedBy }, ...]
  // #endregion example
  return members;
}

File: examples/sharing/group-members.swift
import JsBaoClient

// Current members of a group.
func listGroupMembers(
  client: JsBaoClient,
  groupType: String,
  groupId: String
) async throws {
  // #region example
  let members = try await client.groups.listMembers(groupType: groupType, groupId: groupId)
  // { items: [{ userId, userName, userEmail, addedAt, addedBy }, ...] }
  // #endregion example
  _ = members
}
